liblll/CompilerState.cpp

```diff
@@ -54,8 +54,8 @@ void CompilerState::populateStandard()
 	"(def 'msg (gaslimit to value data) { [0]:data (msg gaslimit to value 0 32) })"
 	"(def 'msg (to value data) { [0]:data (msg allgas to value 0 32) })"
 	"(def 'msg (to data) { [0]:data (msg allgas to 0 0 32) })"
-	"(def 'create (value code) { [0]:(msize) (create value @0 (lll code @0)) })"
-	"(def 'create (code) { [0]:(msize) (create 0 @0 (lll code @0)) })"
+	"(def 'create (value code) { [0]:{[0]:0xff (msize)} (create value @0 (lll code @0)) })"
+	"(def 'create (code) { [0]:{[0]:0xff (msize)} (create 0 @0 (lll code @0)) })"
```

Comment:
> I think I shouldn't review stuff late.

No problem :-)  - this one was really tricky to hunt down.

> Also by looking at LLL, isn't alloc supposed to do the above in a non-hacky way?

I'll have a look at `alloc` in the a.m. Past my bedtime too...